spec/models/category_localization_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+describe "CategoryLocalization" do
+  context "when commit" do
+    it "clears the cache for the locale" do
+      category = Fabricate(:category, name: "yy")
+
+      I18n.locale = "es"
+      expect(Site.all_categories_cache.pluck(:name)).to include("yy")
+
+      I18n.locale = "en"
+      expect(Site.all_categories_cache.pluck(:name)).to include("yy")
+
+      category.update_columns(name: "zz")
+
+      I18n.locale = "es"
+      expect(Site.all_categories_cache.pluck(:name)).to include("yy")
+      I18n.locale = "en"
+      expect(Site.all_categories_cache.pluck(:name)).to include("yy")
+
+      Fabricate(:category_localization, name: "Japón", locale: "es", category:)
+
+      I18n.locale = "es"
+      expect(Site.all_categories_cache.pluck(:name)).to include("zz")
```

Comment:
Shouldn't we be asserting for the name to be `Japón` since we are using the `es` locale and a category localization exists.

Reply:
Oo icic the experimental site setting isn't turned on. My bad.